app/src/main/java/fr/free/nrw/commons/Urls.kt

```diff
@@ -7,6 +7,9 @@ internal object Urls {
     const val CREDITS_URL = "https://github.com/commons-app/apps-android-commons/blob/master/CREDITS"
     const val FAQ_URL = "https://github.com/commons-app/apps-android-commons/wiki/Frequently-Asked-Questions"
     const val PLAY_STORE_URL = "https://play.google.com/store/apps/details?id=fr.free.nrw.commons"
+    const val PLAY_STORE_PREFIX = "market://details?id="
+    const val PLAY_STORE_URL_PREFIX = "https://play.google.com/store/apps/details?id="
+    const val PLAY_STORE_PACKAGE_NAME = "com.android.vending"
```

Comment:
Why is the package name `com.android.vending`?

Reply:
@madhurgupta10 Can you address this comment?

Reply:
@maskaravivek I am unable to extract the PLAY STORE package name using context.
```
InstrumentationRegistry.getInstrumentation().getTargetContext()
```
only gives the app package name.

Reply:
@macgills Could you please suggest an alternative to Hardcoded Play Store Package name?

Reply:
This is fine, that is the package of the play store or at least so my memory tells me

Reply:
But also I don't know why you are using it? Does aboutActivity send an intent with this constant in it? I don't see an update to AboutActivity to use this constant

Reply:
@macgills I am using it because I need to check if the app actually opens the play store or not.

Reply:
I bet you don't need it
```
        Intents.intended(CoreMatchers.anyOf(IntentMatchers.toPackage(Urls.PLAY_STORE_PACKAGE_NAME),
                IntentMatchers.hasData("${Urls.PLAY_STORE_URL_PREFIX}$appPackageName")))
```

The intent doesn't magically get filled with a package we don't put in it, `anyOf` is just masking it. Delete `hasData` and I would say this will fail

Reply:
```
anyOf is just masking it. Delete hasData and I would say this will fail
```
@macgills The test passes for me when I removed the hasData and tested it with only toPackage

Reply:
That is very odd. Ahhh I see my mistake

>Matches an intent based on the package of activity which can handle the intent.

I wouldn't bother testing this, I would more assert that we have the right parameters. This is more useful in an e2e black box test scenario. Harcdoding another app's package also makes me nervous

Reply:
@macgills Okay I will remove this also from Facebook Test!